shipyard_airflow/control/middleware.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2017 AT&T Intellectual Property.  All other rights reserved.
```

Comment:
@eanylin @sh8121att Presumably this is a place holder until keystone integration?